recipes/nbtutor/build.sh

```diff
@@ -0,0 +1,19 @@
+conda install --yes nodejs
+# FIXME: This is an ugly hack. nodejs only gets installed and linked in the
+# first build env and doesn't get linked in following build envs
```

Comment:
Yeah, I get why the hack is there. I am curious as to what the fix is.  Is this reported in another issue? I am happy to merge this in, but I think we shouldn't loose track of this problem.